src/components/30-organisms/carousel/index.scss

```diff
@@ -0,0 +1,88 @@
+@mixin defaultAttributesChildElements {
+  @include typo(primary, large, default);
+  display: flex;
+  text-align: center;
+  justify-content: center; // align the dom element itself (f.e. small spans)
+}
+
+.o-carousel {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 5px; // for arrows
+  height: 100px; // set initial height because of IE bug with align-items and min-height
+  overflow: hidden;
+
+  :focus {
+    outline: none;
+  }
+
+  &__wrapper {
+    margin: 0 10px;
+
+    @include breakpoint($mediaquery-sm-up) {
+      margin: 0 30px;
+    }
+
+    ::slotted(*) {
```

Comment:
no slotted as its very shaky and have the lowest specificity therefore this code wont work as soon as included somewhere.Please use the `inlineStyles` approach